syntax/format_test.go

```diff
@@ -1,77 +1,67 @@
 package syntax
 
 import (
-	"fmt"
-	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
 func TestFmtPrettyForDict(t *testing.T) { //nolint:dupl
 	t.Parallel()
-	simpleDict, err := EvaluateExpr(".", `{'a':1, 'b':2, 'c':3}`)
+	simpleDict, err := EvaluateExpr(".", `{'c':3, 'a':1, 'b':2}`)
 	assert.Nil(t, err)
-	str := FormatString(simpleDict, 0)
-	assert.True(t, contains(str, "\n%sa: 1", 1))
-	assert.True(t, contains(str, "\n%sb: 2", 1))
-	assert.True(t, contains(str, "\n%sc: 3", 1))
+	str, err := FormatString(simpleDict, 0)
+	assert.Nil(t, err)
+	assert.Equal(t, "{\n  a: 1,\n  b: 2,\n  c: 3\n}", str)
```

Comment:
Much nicer :). Nit: Does it still work if you format the expected string like this (much easier to read):
```suggestion
	assert.Equal(t, `
{
  a: 1,
  b: 2,
  c: 3
}`, str)
```